app/Global.scala

```diff
@@ -0,0 +1,34 @@
+import de.zalando.play.controllers.PlayBodyParsing
+import play.api._
+import play.api.mvc.Results._
+import play.api.mvc._
+
+import scala.concurrent.Future
+
+/**
+  * The purpose of this object is to override default play's error reporting with application/json content type.
+  */
+object Global extends GlobalSettings {
```

Comment:
Maybe not _technically_ deprecated, but it's the goal to remove the global state from Play :)
https://www.playframework.com/documentation/2.4.x/Highlights24#motivation